### PR TITLE
Add Malte and Aschaffenburg brandings

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

This adds the option to brand the CMS for Malte and Aschaffenburg.
In the long run, we might also add the possibility to set completely custom logos and themes (see #1093), but this is a first start to support a fixed set of inbuilt themes.

### Proposed changes

- Add option to support different brandings
- Use svg logos for Integreat instead of jpg & png (only use png for PDF export)
- Add logos and accent colors for Malte and Aschaffenburg

You can test the different logos by editing your local config file in `/etc/integreat-cms.ini` or set the environment variable `export INTEGREAT_CMS_BRANDING=malte` or `export INTEGREAT_CMS_BRANDING=aschaffenburg` before starting the dev server.

### Resolved issues

Fixes partially: #1093

### CODE REVIEW

**Comment by svenseeberg** [comment]
I think we should add a note to our README file that the license does not cover logos in this repository.

**Comment by timoludwig** [comment]
> I think we should add a note to our README file that the license does not cover logos in this repository.

Good point! I was unsure about this because we're also not doing this in our [other repos](https://github.com/digitalfabrik/integreat-app/tree/main/build-configs/aschaffenburg/assets) and I thought it might be covered by this section of the Apache license:

>    6. Trademarks. This License does not grant permission to use the trade
      names, trademarks, service marks, or product names of the Licensor,
      except as required for reasonable and customary use in describing the
      origin of the Work and reproducing the content of the NOTICE file.

Or isn't this sufficient when the logos aren't registered trademarks? Do we need a different license file for the logos, or is it just sufficient to exclude them from the general license, leaving them "without license"?

File: README.md [hunk 1]
```diff
@@ -48,3 +48,10 @@ cd integreat-cms
 For detailed instructions, tutorials and the source code reference have a look at our great documentation:
 
 <p align="center">:notebook: https://digitalfabrik.github.io/integreat-cms/</p>
+
+
+## License
+
+This project is licensed under the Apache 2.0 License, see [LICENSE.txt](./LICENSE.txt)
+
+All files in [./integreat_cms/static/src/logos/](./integreat_cms/static/src/logos/) are not covered by this license and may only be used with specific permission of the copyright holder.
```
Comment (timoludwig) [inline README.md:57]: @svenseeberg Do you think this is sufficient?

Comment (svenseeberg) [reply README.md:57]: Sounds good, thanks!

**Review by ulliholtgrave — APPROVED** [review]
I tried all variants and the mentioned functions. Looks good to me and thanks for your effort! :+1:

**Pull request merged**